app/components/optics/sidebar/component.rb

```diff
@@ -41,30 +41,22 @@ def variant_class
       class Brand < ApplicationViewComponent
         accepts :url
         accepts :image_source
+        accepts :name
```

Comment:
Is name for if you don't have an image and just want text? If so, that a nice feature.

Might change `url` to `href` to be closer to how HTML is used. Might also change `image_source` to `img_src`. Normally we try not to abbreviate, but in this case I think it reflects the actual underlying html and would be more familiar to both devs and designer who have done html.

Reply:
I'm not sure about changing `url` to `href`. Rails method signatures use `url`. Plus it's not a straightforward 1:1 mapping. If you pass `url` to Button it becomes an `<a>`, otherwise it's `<button>`. I'm fine with `img_src`.